rust_snuba/benches/processors.rs

```diff
@@ -153,6 +153,7 @@ fn create_factory(
         concurrency,
         None,
         true,
+        None,
     );
```

Comment:
I think this `new` function is a candidate for refactoring with a `StrategyFactoryOptions` struct or something (not necessarily in this PR).